www/artists/emmalake/gallery.html

```diff
@@ -22,34 +22,53 @@ <h1>
                     <table border="0" width="75%">
                         <tr>
                             <td width="20%">
-                                <a id="contentlink" href="../amckay/1.jpg"><img id="thumbnail" src="a_mckay.jpeg" alt="no alt text"></a>
+                                Art McKay
+                                <br>
+                                <a id="contentlink" href="https://www.saskartists.ca/artists/amckay/index.html"><img id="thumbnail" src="a_mckay.jpeg" alt="no alt text"></a>
                             </td>
                             <td width="20%">
+                                "Flux"
+                                <br>
                                 <img src="flux.jpeg" alt="no alt text">
                             </td>
                             <td width="20%">
+                                Untitled
+                                <br>
                                 <img src="round_art.jpeg" alt="no alt text">
                             </td>
                             <td width="20%">
+                                His Rectangular art
+                                <br>
+                                </article>
```

Comment:
I think this is the tag that's causing the HTML validator to glitch out. I don't see an opening article tag, particularly not at the level of this article closing tag. Remove this, and the full validator should be able to run - it's currently crashing after this error.